src/sentry/api/helpers/group_index/delete.py

```diff
@@ -74,7 +75,7 @@ def delete_group_list(
             "transaction_id": transaction_id,
             "eventstream_state": eventstream_state,
         },
-        countdown=3600,
+        countdown=countdown,
```

Comment:
The original countdown value comes from here:
https://github.com/getsentry/sentry/commit/951e993b56cc6ab768ea0883d73d9faefd065d08